README.md

```diff
@@ -2,147 +2,142 @@
 
 ## Introduction
-`torchao` is a PyTorch library for quantization and sparsity.
+
+torchao is a library to create and integrate high-performance custom data types, layouts and kernels into their PyTorch workflows with up to
+* **30% speedups** for [training](#training)
+* **2x speedups** with **65%** less VRAM for [inference](#inference)
+
+All with no intrusive code changes and minimal accuracy degradation.
+
+## Benchmarks
+
+### Inference
+
+#### Without intrusive code changes
+
+Quantizing your models is a 1 liner that should work on any model with `nn.Linear` including your favorite HuggingFace model. You can find a more comprehensive usage instructions [here](torchao/quantization/) and a hugginface inference example [here](scripts/hf_eval.py)
+
+```python
+from torchao.quantization.quant_api import quantize
+m = quantize(m, "int4wo")
+```
+
+Benchmarks are run on a machine with a single A100 GPU using the script in `_models/llama` which generates text in a latency-optimized way (batchsize=1)
+
+The models used were `meta-llama/Llama-2-7b-chat-hf` and `meta-llama/Meta-Llama-3-8B`.
+
+| Model       | Technique          | wikitext-perplexity | Tokens/Second | Memory Bandwidth (GB/s) | Peak Memory (GB) | Model Size (GB) |
+| ----------- | ------------------ | ------------------- | ------------- | ----------------------- | ---------------- | --------------- |
+| Llama-2-7B  | Base (bfloat16)    | 12.212              |  105.02       | 1387.78                 | 13.21            | 13.90           |
+|             | int8dq             | 12.262              |  9.40         | 62.26                   | 6.62             | 8.61            |
+|             | int8wo             | 12.204              |  147.03       | 973.54                  | 6.62             | 8.95            |
+|             | int4wo-64          | 12.843              |  199.81       | 746.45                  | 3.74             | 4.75            |
+|             | int4wo-64-GPTQ     | 12.489              |  199.81       | 746.45                  | 3.74             | 4.75            |
+| Llama-3-8B  | Base (bfloat16)    |                  |  94.91        | 1424.58                 | 15.01            | 16.43           |
+|             | int8dq             |                  |  8.41         | 63.23                   | 7.52             | 9.24            |
+|             | int8wo             |                  |  136.75       | 1028.38                 | 7.52             | 10.42           |
+|             | int4wo-64          |                  |  179.41       | 757.45                  | 4.22             | 6.88            |
+
+note: Int8 dynamic quantization works best on compute bound models like [SAM](https://github.com/pytorch-labs/segment-anything-fast) whereas Llama with batchsize=1 tends to be memory bound, thus the rather low performance.
```

Comment:
I thought it was more about compute bound vs memory bound, and less about SAM vs LLaMa

Reply:
indeed that's intent of the phrase but seems like it's not as clear as I'd thought

Reply:
better?